llm/prompt/template/openai/chat.py

```diff
@@ -38,3 +39,65 @@ def get_stylish_completion(prompt, style="Passionate and friendly"):
     messages = prompt_template.format_messages(style=style, prompt=prompt)
     response = chat(messages)
     return response.content
+
+
+def llmChain(prompt_input, chain_input):
+    llm = ChatOpenAI(temperature=0.9, model=llm_model)
+    prompt = ChatPromptTemplate.from_template(prompt_input)
+    chain = LLMChain(llm=llm, prompt=prompt)
+
+    return chain.run(chain_input)
+
+
+def simpleSequentialChain(prompt1, prompt2, chain_input):
```

Comment:
consider using *args to make this function more reuable

Reply:
Great suggestion! I've reworked it using a list. Give it a look and tell me if it works for you. Thanks!